lib/screens/create_wallet/create_wallet_screen.dart

```diff
@@ -15,59 +18,148 @@ import 'xprv_card.dart';
 class CreateWalletScreen extends StatefulWidget {
   static final route = '/create_wallet';
   @override
-  _CreateWalletScreenState createState() => _CreateWalletScreenState();
+  CreateWalletScreenState createState() => CreateWalletScreenState();
 }
 
-class _CreateWalletScreenState extends State<CreateWalletScreen> {
+class CreateWalletScreenState extends State<CreateWalletScreen> {
   dynamic currentFormCard;
+  dynamic nextAction;
+  dynamic secondaryAction;
+  dynamic prevAction;
+  double bottomSize = 80;
+
+  List<Widget> _actions() {
+    List<Widget> actions = [
+      PaddedButton(
+          padding: EdgeInsets.only(bottom: 8),
+          text: nextAction != null ? nextAction().label : 'Continue',
```

Comment:
Are these equivalent?
```suggestion
          text: nextAction()?.label ?? 'Continue',
```

Reply:
Next action can be null and therefore cannot be called before checking